src/ConnectedMode/Suppressions/ServerIssuesStore.cs

```diff
@@ -19,24 +19,59 @@
  */
 
 using System;
-using System.Linq;
 using System.Collections.Generic;
 using System.ComponentModel.Composition;
+using System.Linq;
 using SonarQube.Client.Models;
 
 namespace SonarLint.VisualStudio.ConnectedMode.Suppressions
 {
     [Export(typeof(IServerIssuesStore))]
+    [Export(typeof(IServerIssuesStoreWriter))]
     [PartCreationPolicy(CreationPolicy.Shared)]
-    internal class ServerIssuesStore : IServerIssuesStore
+    internal class ServerIssuesStore : IServerIssuesStore, IServerIssuesStoreWriter
     {
-        private IEnumerable<SonarQubeIssue> serverIssues;
+        private List<SonarQubeIssue> serverIssues;
 
         public event EventHandler ServerIssuesChanged;
 
+        #region IServerIssuesStore implementation
```

Comment:
I added the #region thing because it made sense to me this way its easier to know which part is which but I can also remove it if you think thats better

Reply:
I use `#region` in the same way, to separate interfaces implementations. An alternative is to make it a `partial` class and put the implementations in different files.

Rita wasn't a big fan of regions because she had seen them being used badly previously.